config/routes.rb

```diff
@@ -3,9 +3,12 @@
   get '/login', to: 'session#new'
   post '/login', to: 'session#create'
   delete '/logout', to: 'session#destroy'
-  resources :users, only: [:new, :index, :create, :show]
+  get '/dashboard', to: 'users#show'
+  resources :users, only: [:new, :index, :create]
   resources :items, only: [:index]
   resources :categories, only: [:index]
   get '/categories/:slug', to: 'categories#show', as: "category"
   resources :cart_items, only: [:create, :show, :index, :update, :destroy]
+  resources :orders, only: [:create]
+  get '/orders', to: 'orders#show'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.